chapter_01/exercises/Chap1_projectile_motion.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2019 Cheick Bamba
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+# The formula of projectile motion
```

Comment:
Good job with the comments! You may also want to take a look at https://realpython.com/documenting-python-code/ for guidelines on Python code documentation.